x-pack/plugins/lens/public/visualizations/xy/xy_suggestions.ts

```diff
@@ -587,6 +587,7 @@ function buildSuggestion({
         // Don't advertise charts without at least one split
         (!xValue && !splitBy)),
     state,
+    incomplete: yValues.length === 0,
```

Comment:
nit: I would define an `isIncomplete` before the return and reuse it here and for the `hide`, so in the future we won't forget to sync the two.

Reply:
Fixed https://github.com/elastic/kibana/pull/153931/commits/fd12a4654c7624cafd1e955555f06334ad55df81